src/edid.c

```diff
@@ -22,13 +22,93 @@
 
 /* Author: Soren Sandmann <sandmann@redhat.com> */
 
+#include <stdio.h>
 #include <errno.h>
 #include <stdlib.h>
 #include <string.h>
 #include <math.h>
 
 #include "edid.h"
 
+#define TAB_LEN 8
+#define KEYWORD_LEN 13 // Max length of longest keyword (Currently Aspect Ratio)
+
+/* TODO: Turn these print_edid_* functions into macros? Horrible code duplication! :/ */
+int print_edid_heading(const char *prefix, const char *data, const char *postfix, int level)
+{
+	if (data) {
+		printf("%-*s", (level * TAB_LEN), "");
+		printf("%s%s%s",
+		       prefix, (strlen(data) > 0) ? data : "N/A", postfix);
+	}
+	else {
```

Comment:
We use Linux style, so the `else` should be "cuddled" like this: `} else {`, please fix this and the others below (like you did last in the very same file :-)